packages/roosterjs-content-model-dom/package.json

```diff
@@ -3,7 +3,8 @@
     "description": "Content Model for roosterjs",
     "dependencies": {
         "tslib": "^2.3.1",
-        "roosterjs-content-model-types": ""
+        "roosterjs-content-model-types": "",
+        "roosterjs-editor-types": ""
```

Comment:
do not add dependency to old packages. Content model code is brand new, it has no dependency to old code. Old code will be deleted